### PR TITLE
refactor-index-pages

### PR DESCRIPTION
**Pull request by daothanhcam**

### CODE REVIEW

File: app/views/admin/skills/index.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <% provide :title, t("skill.index.title") %>
 <h1><%= t("skill.index.title") %></h1>
```
Comment (phonghenry14) [inline app/views/admin/skills/index.html.erb]: bor ()

File: app/views/admin/skills/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-<p><%= link_to "Create New Skill", new_admin_skill_path, class: "btn btn-success" %></p>
-<%= will_paginate %>
+<p><%= link_to "Create New Skill", new_admin_skill_path,
+  class: "btn btn-success" %></p>
 
 <table>
   <tr><th><%= t("skill.index.name") %><th></tr>
```
Comment (phonghenry14) [inline app/views/admin/skills/index.html.erb]: Bỏ

File: app/views/admin/skills/index.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <% provide :title, t("skill.index.title") %>
 <h1><%= t("skill.index.title") %></h1>
-<p><%= link_to "Create New Skill", new_admin_skill_path, class: "btn btn-success" %></p>
-<%= will_paginate %>
+<p><%= link_to "Create New Skill", new_admin_skill_path,
```
Comment (khanhkhanh) [inline app/views/admin/skills/index.html.erb]: i18n

File: app/views/admin/skills/index.html.erb (hunk 1 above)
Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: try remove ()

File: app/views/admin/skills/index.html.erb (hunk 2 above)
Comment (khanhkhanh) [inline app/views/admin/skills/index.html.erb]: Remove ()

File: app/views/admin/skills/index.html.erb (hunk 3 above)
Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: i18n?

File: app/views/admin/skills/index.html.erb (hunk 2 above)
Comment (vjnhat) [inline app/views/admin/skills/index.html.erb]: try remove ()

File: app/views/admin/users/index.html.erb [hunk 4]
```diff
@@ -1,21 +1,11 @@
 <% provide :title, t("user.all_user") %>
 <p>
-  <h1>t "user.all_user"<br>
+  <h1><%= t "user.all_user" %>
```
Comment (khanhkhanh) [inline app/views/admin/users/index.html.erb]: users.all.count => users.count

File: app/views/admin/users/index.html.erb [hunk 5]
```diff
@@ -1,21 +1,11 @@
...
   (<%= @users.all.count %>)</h1>
```
Comment (khanhkhanh) [inline app/views/admin/users/index.html.erb]: users.all.count => users.count

**Pull request merged**